src/adam.cpp

```diff
@@ -35,7 +35,7 @@
 
 namespace miopen {
 
-miopenStatus_t Adam(Handle& handle,
+miopenStatus_t Adam(Handle const& handle,
```

Comment:
If you look at the other parameters, they are all defined using "const type&" rather than "type const&".  Can we please keep this consistent and use "const type&" in all locations.  Hopefully this can just be a quick find/replace on "Handle const&" to "const Handle&".